bundle/src/main/java/com/adobe/acs/commons/httpcache/engine/impl/HttpCacheEngineImpl.java

```diff
@@ -399,7 +399,7 @@ public boolean deliverCacheContent(SlingHttpServletRequest request, SlingHttpSer
 
         // Copy the cached data into the servlet output stream.
         try {
-            IOUtils.copy(cacheContent.getInputDataStream(), response.getOutputStream());
+            IOUtils.copy(cacheContent.getInputDataStream(), response.getWriter());
```

Comment:
Saw this in your commit, I had it temporarily to getWriter because it caused an error on my part :/ an exception occured when writing to the outputstream. Not sure how to make it work with it

Reply:
java.lang.IllegalStateException: null
	at org.apache.sling.scripting.jsp.util.JspSlingHttpServletResponseWrapper.getOutputStream(JspSlingHttpServletResponseWrapper.java:79)
	at com.adobe.acs.commons.httpcache.engine.impl.HttpCacheEngineImpl.deliverCacheContent(HttpCacheEngineImpl.java:401)


I cannot seem to workaround this one , pretty nasty